tn-alloc/tn-alloc.schema.json

```diff
@@ -0,0 +1,256 @@
+            "d": {
+              "description": "Attributes block SAID",
+              "type": "string"
+            },
+            "u": {
+              "description": "A salty nonce",
+              "type": "string"
+            },
+            "i": {
+              "description": "Recipient Brand AID",
+              "type": "string"
+            },
+            "dt": {
+              "description": "Issuance date time",
+              "format": "date-time",
+              "type": "string"
+            },
+            "numbers": {
+              "description": "Telephone Numbers allocated to enterpise either as a list of specific numbers or a range of numbers",
+              "type": "object",
+              "properties": {
+                "tn": {
+                  "description": "Specific telephone numbers list",
+                  "type": "array",
+                  "uniqueItems": true,
+                  "minItems": 1,
+                  "items": {
+                    "type": "string",
+                    "description": "Telephone number in E164 format",
+                    "pattern": "[+][0-9]{7,15}"
+                  }
+                },
+                "rangeStart": {
+                  "description": "Range start number in E164 format, inclusive of this number. If provided, rangeEnd must also be provided",
+                  "type": "string"
+                },
+                "rangeEnd": {
+                  "description": "Range end number in E164 format, inclusive of this number. If provided, rangeStart must also be provided",
+                  "type": "string"
+                }
+              },
+              "additionalProperties": false
+            },
+            "channel": {
+              "description": "Communication channel for which number will be used (e.g., sms, voice, etc.)",
+              "type": "string",
+              "pattern": "sms|voice"
+            },
+            "doNotOriginate": {
+              "description": "True or False based on condition if telephone number is a 'do not originate' number (inbound only)",
+              "type": "boolean"
+            },
+            "startDate": {
+              "description": "Start date",
+              "format": "date-time",
+              "type": "string"
+            },
+            "endDate": {
+              "description": "Expiration date",
+              "format": "date-time",
+              "type": "string"
+            }
+          },
+          "additionalProperties": false
+        }
+      ]
+    },
+    "e": {
+      "oneOf": [
+        {
+          "description": "Edges block SAID",
+          "type": "string"
+        },
+        {
+          "$id": "ED-ETRO9FOiEgvK8P66Rut-Cun-0aZ5FbgTlngPf9yVT",
+          "description": "Edges block",
+          "type": "object",
+          "properties": {
+            "d": {
+              "description": "Edges block SAID",
+              "type": "string"
+            },
+            "tnalloc": {
```

Comment:
I think tn-alloc also needs group auth as edge as well.

Reply:
@khagesh there is another optional node `issuer` in `tn-alloc` schema and there is no schema restriction on this node. Whenever needed we can use this to chain LE or group-auth or another credential.